api/src/main/java/org/eclipse/microprofile/openapi/annotations/extensions/package-info.java

```diff
@@ -15,6 +15,6 @@
  * Annotations to represent additional data to extend existing OpenAPI specifications.
  */
 
-@org.osgi.annotation.versioning.Version("1.0")
+@org.osgi.annotation.versioning.Version("1.1")
```

Comment:
This is to fix an error introduced in 2.0-MR1. Since the version was treated as a final and not a release candidate as the naming conversion for releasing candidates is RC or Mx not MRx. This current version is different from the 2.0-MR1, which was treated as the current final release. Hence, we have to increase the minor version to bypass the issue.

Reply:
:+1: Thanks